cfde_submit/config.py

```diff
@@ -35,14 +36,17 @@
         },
         "handlers": {
             "console": {
-                "class": ("logging.StreamHandler" if os.getenv("CFDE_SUBMIT_LOGGING")
-                          else "logging.NullHandler"),
-                "level": os.getenv("CFDE_SUBMIT_LOGGING") or "INFO",
+                "class": "logging.StreamHandler",
                 "formatter": "basic",
+                "level": "NOTSET"
             }
         },
         "loggers": {
-            "cfde_submit": {"level": "DEBUG", "handlers": ["console"]},
+            'cfde_submit': {
+                "propagate": False,
+                "level": "NOTSET",
+                "handlers": ["console"],
+            },
```

Comment:
In the future, it might be good to have different handlers for log messages: One for developers that provides more diagnostic information, and another for users that is cleaner and just adds more information about what the cfde-submit client is doing.